### DISCUSSION TITLE
How to remove the space reserved for FormItem Help / _validationMessages?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2428] [Q&A]
I am using CSLA for my domain models which supports errors, warnings and information messages when validating properties. After the edit control in a `FormItem` I am displaying the messages with appropriate colours.

```html
<FormItem Label          = "@(IsLabelVisible ? @Property.FriendlyName : null)"
          Required       = "@Required"
          ValidateStatus = "@ValidateStatus"
          @ref           = "_formItem">
    @EditControl
    <div>
        <Text Type = "danger"   >@Property.ErrorText</Text>
        <Text Type = "warning"  >@Property.WarningText</Text>
        <Text Type = "secondary">@Property.InformationText</Text>
    </div>
</FormItem>
```

The `FormItem` reserves space for the `Help` or `_validationMessages` which I won't be using. Is there a way to remove this extra space, show in magenta below?

![image](https://user-images.githubusercontent.com/21806535/167340505-9f474f94-9150-4b08-b631-edba656c65ec.png)

**Comment by ElderJames** [comment]
Will fix in next release.
